### PR TITLE
Footer 

### PR DESCRIPTION
**Pull request by wildcoder51**
Add characters counter (it works)

### CODE REVIEW

**Comment by rocambille** [comment]
there are conflicts to resolve in package.json

**Comment by XeniaR1S** [comment]
It's done

File: markode/package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.4.0",
     "@testing-library/user-event": "^7.2.1",
+    "node-modules": "^1.0.1",
```
Comment (rocambille) [inline markode/package.json]: according to [documentation](https://www.npmjs.com/package/node-modules), this package is intended to be installed globally. are you sure you want to integrate it in the project?

File: markode/src/App.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
-import Main from './components/Main';
+// import Main from './components/Main';
```
Comment (rocambille) [inline markode/src/App.js]: dead code: remove it

File: markode/src/components/AllFooter.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+
+  countChar(event){
+    this.setState({numChar: event.target.value.length});
+  }
+  
+  render() {
+    return (
+     <div>
```
Comment (rocambille) [inline markode/src/components/AllFooter.jsx]: the div is useless here: remove it or use a fragment

File: markode/src/components/Counter.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+class Counter extends React.Component{
+    render() {
+      return (
+        <div id="counter">
+          <p>You inserted {this.props.numChar} characters</p>
```
Comment (rocambille) [inline markode/src/components/Counter.jsx]: can you put the id on the paragraph? if yes, then the div is useless

File: markode/src/components/CounterText.jsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+class CounterText extends React.Component{
+    render() {
+      return (
+        <div id="counterText">
+            <textarea placeholder="Ecris ton markdown ici" onChange={this.props.countChar}/>
```
Comment (rocambille) [inline markode/src/components/CounterText.jsx]: can you put the id on the paragraph? if yes, then the div is useless

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by XeniaR1S** [comment]
I make the changes

**Pull request merged**